src/libs/ReportUtils.ts

```diff
@@ -5417,6 +5418,40 @@ function buildOptimisticExpenseReport(
     return expenseReport;
 }
 
+function buildOptimisticEmptyReport(reportID: string, accountID: number, parentReportID: string | undefined, policy: OnyxEntry<Policy>, timeOfCreation: string) {
+    const {stateNum, statusNum} = getExpenseReportStateAndStatus(policy);
+    const titleReportField = getTitleReportField(getReportFieldsByPolicyID(policy?.id) ?? {});
+    const optimisticEmptyReport: OptimisticNewReport = {
+        reportName: '',
+        reportID,
+        policyID: policy?.id,
+        type: CONST.REPORT.TYPE.EXPENSE,
+        currency: policy?.outputCurrency,
+        ownerAccountID: accountID,
+        stateNum,
+        statusNum,
+        total: 0,
+        nonReimbursableTotal: 0,
+        participants: {},
```

Comment:
Maybe declare it earlier like this:

```
const participants = accountId ?? {
    [accountID]: {
        notificationPreference: CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
    },
} : {};
```
`optimisticEmptyReport.ownerAccountID = accountID`; 
is declared twice, so we can omit it in the` if (accountID)`, and as a result, the entire if can be removed.